x-pack/plugins/observability_solution/infra/common/locators/locators.test.ts

```diff
@@ -17,9 +17,10 @@ import {
   LogsLocatorParams,
   NodeLogsLocatorParams,
 } from '@kbn/logs-shared-plugin/common';
+import { InfraLocatorDependencies } from '.';
```

Comment:
Q: Do we need those changes in `infra` after the locators are moved to the `observability_shared`?